stable/voyager/voyager/templates/deployment.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    chart: {{.Chart.Name}}-{{.Chart.Version}}
+    heritage: {{.Release.Service}}
+    release: {{.Release.Name}}
+    run: {{ template "fullname" . }}
+  name: {{ template "fullname" . }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      run: {{ template "fullname" . }}
+  template:
+    metadata:
+      labels:
+        run: {{ template "fullname" . }}
+    spec:
+      containers:
+      - args:
+        - --cloud-provider={{.Values.cloudProvider}}
+        - --cluster-name={{.Values.clusterName}}
+        - --v=3
+        image: "{{.Values.image}}:{{.Values.imageTag}}"
+        name: {{ template "name" . }}
+        ports:
+        - containerPort: 1234
+          name: zero
```

Comment:
Are these things (port 1234, name: zero) something related to voyager? If not I would prefer just using 8080 and name: http here or something similar.